docs/configuration/index.md

```diff
@@ -115,7 +115,8 @@ Previously known as `upstream_project_name`, which has been deprecated since Pac
 #### upstream_project_url
 
 (*string*) URL of the upstream git project (e.g.
-`https://github.com/packit/packit`).
+`https://github.com/packit/packit`). This isn't the URL to the upstream project tarball.
+The URL to the upstream project tarball is taken from the specfile when needed.
```

Comment:
This is not really true for upstream jobs. There the tarball is generated and a potential URL in spec file is overwritten with it.

```suggestion
`https://github.com/packit/packit`).

:::info

Do not confuse this with the URL of upstream tarball. In case of `propose_downstream`, `pull_from_upstream`, `koji_build` and `bodhi_update` jobs, upstream tarball URL is taken from spec file or from [sources](#sources).

:::
```